.github/workflows/acc-tests.yaml

```diff
@@ -0,0 +1,52 @@
+# Flow based on proposal from: https://docs.google.com/document/d/1Gz0DohBnmH2WaqrQA3JCix_m6ISVyX56nYMDdT5worM/edit?usp=sharing
+
+# use like:
+
+# name: Run Acceptance Tests
+# on: push
+# jobs:
+#   run-acceptance-tests:
+#     uses: hashicorp/vault-workflows-common/.github/workflows/acc-tests.yaml
+#     secrets: inherit
+
+name: Run Acceptance Tests
+
+on:
+  workflow_call
+
+jobs:
+  run-acc-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: './go.mod'
+          cache: true
+      - name: Set test environment
+        run: |
+          for v in $(echo '${{ toJson(secrets) }}' | jq -r 'to_entries|map("\(.key)=\(.value|tostring)")|.[]'); do
+              echo $v >> $GITHUB_ENV
+          done
+          make setup-env
+      - name: Set local Vault dev instance
+        run: |
+          make dev
+          export VAULT_ADDR=http://127.0.0.1:8200
+          export VAULT_TOKEN=root
+          git clone https://github.com/hashicorp/vault.git
+          cd vault
+          make dev
+          bin/vault server -dev -dev-root-token-id=root -dev-plugin-dir=../bin 2>&1 > /dev/null &
+          cd ..
+          source ./bootstrap/terraform/local_environment_setup.sh
+          make configure
```

Comment:
I'm wondering if it is better to pull the latest official docker image or try to build the latest version to confirm the plugin is ready for the upcoming release.

Docker pull would be faster and a broken Vault main branch would not cascade to plugin CI failures. But testing against main lets us catch problems faster.

Reply:
Why not both?

Reply:
I adjusted the strategy, instead of building from tip of main we run the tests on the latest + 3 minor versions of Vault so we should avoid flakiness from bad merge to main while covering all supported versions.